modules/virt-creating-storage-class.adoc

```diff
@@ -38,24 +40,26 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: hostpath-provisioner <1>
-provisioner: kubevirt.io/hostpath-provisioner
-reclaimPolicy: Delete <2>
-volumeBindingMode: WaitForFirstConsumer <3>
+provisioner: kubevirt.io.hostpath-provisioner <2>
+reclaimPolicy: Delete <3>
+volumeBindingMode: WaitForFirstConsumer <4>
+parameters:
+ storagePool: <any_pool> <5>
 ----
-<1> You can optionally rename the storage class by changing this value.
-<2> The two possible `reclaimPolicy` values are `Delete` and `Retain`. If you
+<1> Assign any meaningful name to the storage class. In this example, `csi` is used to specify that the class is using the CSI provisioner, instead of the legacy provisioner. Choosing descriptive names for storage classes, based on legacy or CSI driver provisioning, eases implementation of your migration strategy.
+<2> The legacy provisioner uses `kubevirt.io/hostpath-provisioner`. The CSI provisioner uses `kubevirt.io.hostpath-provisioner`.
+<3> The two possible `reclaimPolicy` values are `Delete` and `Retain`. If you
 do not specify a value, the storage class defaults to `Delete`.
-<3> The `volumeBindingMode` value determines when dynamic provisioning and volume
-binding occur. Specify `WaitForFirstConsumer` to delay the binding and provisioning
-of a PV until after a pod that uses the persistent volume claim (PVC)
-is created. This ensures that the PV meets the pod's scheduling requirements.
+<4> The `volumeBindingMode` value determines when dynamic provisioning and volume binding occur. Specify `WaitForFirstConsumer` to delay the binding and provisioning of a PV until after a pod that uses the persistent volume claim (PVC) is created. This ensures that the PV meets the pod's scheduling requirements.
+<5> The storage class you create for the CSI driver custom resource specifies additional parameters that indicate what storage pool (`<any_pool>`) is associated with the storage class. `storagePool` is only available for use with the CSI provisioner `kubevirt.io.hostpath-provisioner`.
```

Comment:
Maybe point out that <any_pool> should match the name of a storage pool that is defined in the HPP CR.

Reply:
We've done this.